pom.xml

```diff
@@ -24,8 +24,8 @@
   <version>0.28.2-SNAPSHOT</version>
   <packaging>pom</packaging>
 
-  <name>Nessie</name>
-  <description>Transactional Catalog for Data Lakes</description>
+  <name>Nessie Apprunner</name>
+  <description>Maven and Gradle plugins to use Nessie in integration tests</description>
```

Comment:
should we mention gradle here?

Reply:
It's technically a sub-module of this one, so I suspect that's okay.